include/avif/avif.h

```diff
@@ -846,6 +947,12 @@ typedef struct avifIOStats
     size_t colorOBUSize;
     // Size in bytes of the AV1 image item or track data containing alpha samples.
     size_t alphaOBUSize;
+#if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
+    // Size in bytes of the AV1 image item containing gain map samples.
+    size_t gainMapOBUSize;
```

Comment:
Nit: Remove "OBU" from the member's name. Or perhaps change "OBU" to "Image"?

Reply:
If this is the size of the output of the AV1 encoder (or the input to the AV1 decoder), then it's fine to use "OBU" in its name for consistency.

Note: It would have been more accurate to call it "Bitstream". An AV1 bitstream is divided into _open bitstream units_ (OBUs), so an OBU is just a segment of a bitstream.

Reply:
Removed this field.